### ISSUE TITLE
A11y: spinner issue

### ISSUE THREAD

**Issue by reillz10** [edx/gradebook#32]
"a11y noting the need for exposing the state programmatically, and focus management. And is should announce "ready" to screen reader users when removed. We should make a Paragon component for this. OK to defer a proper fix until that component is created, for this specific case, as it is low-risk UX scenario." ~ Jeff Witt

Considerations
- Focus management
- Text alternatives (and labels)
- 3:1 contrast for the colors
- messages to visually impaired users
   - off!
   - waiting...
   - done!
